.github/workflows/build-docker-image.yml

```diff
@@ -0,0 +1,35 @@
+name: Build Docker image
+
+on:
+  workflow_dispatch:
+    inputs:
+      push:
+        description: 'Push the image to registry?'
+        default: "false"
+        required: false
+      ref:
+        description: 'Branch, PR, tag or commit to build'
+        default: "master"
+        required: false
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.inputs.ref }}-${{ github.event.inputs.push }}
+  cancel-in-progress: true
+
+jobs:
+  bulid_docker_image:
```

Comment:
```suggestion
  build_docker_image:
```

Reply:
I had to look reallly closely to catch that even in your diff.